### ISSUE TITLE
Launch Pads - Launching to Random Sectors

### ISSUE THREAD

**Issue by 50275** [Anuken/Mindustry-Suggestions#2930]
### Describe what you would like changed, and why.

Launch Pads should be able to launch items at random to other sectors, and by default when no region is selected. Currently, launch pads always accept items, destroying them if no area is select, sending them at 5/sec, and their configuration forces all of them to launch to the same sector at the same time.  

Unfortunately, this means that the first few sectors to gain a new type of resource (e.g. thorium) will have no good way to stockpile it, meaning that frequent checks on the tech tree will be necessary. For reference, almost immediately after you get thorium, available thorium-heavy techs are: Vault, Oil Extractor, Nucleus Core, Multiplicative Reconstructor, Thorium Reactor, 
(Large) Thorium Wall. 

In case you're considering in placing it all in less than three sectors, just remember that a foundation core can only have 9000 items each. If I remember correctly, the Multiplicative Reconstructor costs multiple times that. 

### Describe the changes you want to propose. Include possible alternatives.

Allow Launch Pads to send items to random sectors and make it the default setting instead of launching to the sun. 

For example, bringing back the V5 style loadout could work instead. Launching items to there could technically be an option to make frequent checks on the tech tree unnecessary.

**Comment by RealYeet** [comment]
TL;DR make it so that launch pads don’t destroy items put into them when no destination is selected.

**Comment by OnyxFinger** [comment]
how do you even manage to NOT select a destination ?

**Comment by RealYeet** [comment]
When a launch pad is first built or it is launching to a sector that is lost it loses its destination. This is as of 130.1 and launch pads incinerate resources put into them once this happens because it shoots into the  “sun”

**Comment by Ymabert** [comment]
@50275 When there is overproduction rather than wasting the materials launched would be right that they were launched in a random sector that requires at least 1 of the materials launched.

**Comment by 50275** [comment]
@OnyxFinger the problem is how will you know when the target sector was full with that resource

**Comment by Ymabert** [comment]
when it is 100 less materials than the maximum capacity it is considered full.

**Comment by 50275** [comment]
I've never heard of this, and I don't think it's a thing. https://github.com/Anuken/Mindustry/issues/5638#issuecomment-886047594

**Comment by OnyxFinger** [comment]
how do you even manage to NOT select a destination ?

> @OnyxFinger the problem is how will you know when the target sector was full with that resource

generally, I don't even care, since ressource production and export is a constant flow and never runs out

**Comment by RealYeet** [comment]
If you are core leeching your resources like a lot of people things run out very fast
